### DISCUSSION TITLE
Add Sprite Frames property for TextureRect node

### DISCUSSION THREAD

**Discussion by valkyrienyanko** [godotengine/godot-proposals#5439] [GUI]
![image](https://user-images.githubusercontent.com/6277739/190730975-e44ae91e-ed37-4f01-926b-07075936707c.png)

I have a coin using the AnimatedSprite node because I like the Sprite Frames property where I can supply one image file containing a horizontal list of all the sprites.

But using the AnimatedSprite node makes me use a Control node with a horizontal padding value that I have to manually guess.

The TextureRect node would fix this solution but then I would have to specify each individual animation frame and I would have to go into my image editor and separate the coin frames into individual files.

I propose that the TextureRect node gets support for a Sprite Frames property like the one from the AnimatedSprite node.

**Comment by bend-n** [comment]
You dont have to "split into individual files", you just change the `region_rect` of the texture rects sprite.

**Reply by valkyrienyanko** [reply]
![image](https://user-images.githubusercontent.com/6277739/190847129-38f9fcb1-d2a4-4c30-8755-92ef755af034.png)

I can't find this `region_rect` property
